MeshGeoToolsLib/BoundaryElementsSearcher.h

```diff
@@ -56,27 +56,62 @@ class BoundaryElementsSearcher
      */
     std::vector<MeshLib::Element*> const& getBoundaryElements(GeoLib::GeoObject const& geoObj);
 
+    /// A boundary element is generated out of a face of a bulk element. This
+    /// method returns the bulk element ids and the corresponding face ids.
+    /// @param geometry The boundary elements, bulk element ids and  bulk
+    /// element face ids will be extracted for the given geometry (point,
+    /// polyline or surface).
+    /// @return A vector of id pairs will be returned where the first item of
+    /// the pair is the element id and the second item of the pair is the face
+    /// id.
+    std::vector<std::pair<std::size_t, unsigned>> const& getBulkIDs(
```

Comment:
Ah, I found the docu. You might want to use `\copydoc` if it's all the same text.